Tests/Marketplace/marketplace_services.py

```diff
@@ -405,6 +410,29 @@ def _get_author(support_type, author=None):
         else:
             return author
 
+    @staticmethod
+    def _get_certification(support_type, certification=None):
+        """ Returns pack certification.
+
+        In case support type is xsoar, CERTIFIED is returned.
+        In case support is not xsoar but pack_metadata has certification field, certification value will be taken from
+        pack_metadata defined value.
+        Otherwise empty certification value (empty string) will be returned
+
+        Args:
+            support_type (str): support type of pack.
+            certification (str): certification value from pack_metadata, if exists.
+
+        Returns:
+            str: certification value
+        """
+        if support_type == Metadata.XSOAR_SUPPORT:
+            return Metadata.CERTIFIED
+        elif support_type != Metadata.XSOAR_SUPPORT and certification:
+            return certification
+        else:
+            return ""
```

Comment:
@IkaDemisto 
are you sure the correct value here should be empty and not none?
in the server the values are `none`, `verified`, `certified`.
@asafshen WDYT?

Reply:
@yuvalbenshalom `none` or empty string will have same result.

Reply:
@IkaDemisto please send me a screen shot of the result